source/_less/patternfly-site.less

```diff
@@ -1425,7 +1425,7 @@ th {
 }
 
 .page-tabs-header {
-  margin-bottom: (@grid-gutter-width + 5);
+  margin-bottom: 25px;
```

Comment:
Any reason why you dropped the variable here?

Reply:
I believe it was 45px with the variable and that leaves too wide of a gap between the header and the links and made it look odd, so I replaced it to 25px to make the gap less

Reply:
@matthewcarleton should I keep the variable and just do ```(@grid-gutter-width - 15);```?

Reply:
👍,That's a good question for @andresgalante I'm not sure what the approach is with the pf site.